randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -398,9 +399,11 @@ Extra - room_id: [13, 46]
           All of the following:
               Hi-Jump
               After Boss Zazabi Defeated or Wall Jump (Beginner)
+          # Freeze Zoros
```

Comment:
There also needs to be 
```
# Freeze kihunters
After Boss Nettori Defeated and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
```

Reply:
Don't think the Event check is since you have the same requirements for the kihunters as the zoros

Reply:
Wrong, you cannot freeze and jump on the zoros anymore.